PSPDFKitPlugin/PSPDFKitPlugin.m

```diff
@@ -960,58 +959,49 @@ - (NSArray *)allowedMenuActionsAsJSON
 //    return @(_pdfController.configuration.printSharingOptions);
 //}
```

Comment:
Same question about commented code in here.

Reply:
See https://github.com/PSPDFKit/Cordova-iOS/pull/75#discussion_r286891193